### PR TITLE
Moving CMB to School Districts

### PR DESCRIPTION
**Pull request by Fosol**
Mikaela had renamed agencies in production, which resulted in the script for v01.07.00 to fail at moving properties from the Capital Planning Branch to the appropriate school district.

I have re-added the transfer to move properties from the Capital Management Brach to the appropriate school district.

### CODE REVIEW

**Review by goemen — APPROVED** [review]
lgtm

File: backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Chilliwack', 'Abbotsford', 'Rosedale', 'Cultus Lake', 'Sardis', 'Deroche', 'Errock')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 34')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Abbotsford')
```
Comment (agalbrai) [inline backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql:86]: SD 33 also has Abbotsford in the Where, wouldn't this mean no buildings would be put in SD 34?

File: backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Francois Lake', 'Nechako Lakes', 'Bear Lake', 'Tchentlo Lakes', 'Takla Lake', 'Fort Babine', 'Smithers Landing', 'Granisle', 'Topley Landing', 'Babine Lake', 'Topley', 'Chuchi Lake', 'Trembleur Lake', 'Grand Rapids', 'Tachie', 'Pinchi', 'Fort St. James', 'Dog Creek', 'Donald Landing', 'Decker Lake', 'Bulkley Lake', 'Burns Lake', 'Fraser Lake', 'Fort Fraser', 'Vanderhoof', 'Fancois Lake', 'Grassy Plains', 'Tahtsa Lake', 'Whitesail Reach', 'Ootsa Lake', 'Whitesail Lake', 'Eutsuk Lake', 'Cheslatta Lake', 'Newstubb Lake', 'Natalkuz Lake')
+    AND p.[AgencyId] = @CMBId
+
+UPDATE p
+    SET p.[AgencyId] = (SELECT [Id] FROM dbo.[Agencies] WHERE [Code] = 'SD 92')
+FROM dbo.[Buildings] p
+JOIN dbo.[Addresses] a ON p.AddressId = a.Id
+WHERE a.[AdministrativeArea] IN ('Nisga''a', 'Kitwanga', 'Nass Camp', 'New Aiyansh', 'Laxgalts''ap')
```
Comment (devinleighsmith) [inline backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql:415]: 'Nisga''a' and 'Laxgalts''ap' may cause issues

Comment (devinleighsmith) [reply backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql:415]: at least in gitub they appear to use the same chars.

File: backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql (hunk 1 above)
Comment (Fosol) [reply backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql:86]: Nice catch

File: backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql (hunk 2 above)
Comment (Fosol) [reply backend/dal/Migrations/v01.08.01/Up/PreUp/01-Buildings.sql:415]: The double single quotes are an escape and don't cause any known issues.

**Pull request merged**